internal/session/repository_session_test.go

```diff
@@ -1206,22 +1207,6 @@ func TestRepository_CancelSessionViaFKNull(t *testing.T) {
 				}
 			}(),
 		},
-		{
-			name: "Scope",
-			cancelFk: func() cancelFk {
-				s := setupFn()
-
-				t := &iam.Scope{
-					Scope: &iamStore.Scope{
-						PublicId: s.ProjectId,
-					},
-				}
-				return cancelFk{
-					s:      s,
-					fkType: t,
-				}
-			}(),
-		},
```

Comment:
I removed this test as it was no longer possible to delete a scope without a foreign key reference error from the key_id column in the session when trying to cascade delete the keys in the scope. This limitation is in place to avoid deleting keys that are still encrypting data.